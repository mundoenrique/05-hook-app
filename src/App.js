import CounterApp from './components/01-usestate/CounterApp';
import CounterWithCustomHook from './components/01-usestate/CounterWithCustomHook';
import FormWithCustomHook from './components/02-useeffect/FormWithCustomHook';
import SimpleFrom from './components/02-useeffect/SimpleFrom';
import MultipleCustomHooks from './components/03-examples/MultipleCustomHooks';

function App() {
	return (
		<>
			<h1>Uso de hooks ReactJs</h1>
			<br />
			<hr />
			<h2>useState</h2>
			<br />
			<CounterApp />
			<br />
			<CounterWithCustomHook />
			<br />
			<hr />
			<h2>useEffect</h2>
			<br />
			<SimpleFrom />
			<br />
			<FormWithCustomHook />
			<br />
			<hr />
			<h2>UseFetch</h2>
			<br />
			<MultipleCustomHooks />
		</>
	);
}

export default App;
