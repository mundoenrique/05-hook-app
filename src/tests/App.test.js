import { shallow } from 'enzyme';
import App from '../App';

describe('Prueba en componente <App />', () => {
	test('Debe mostrarse correctamente el componene <App />', () => {
		const wrapper = shallow(<App />);

		expect(wrapper).toMatchSnapshot();
	});
});
