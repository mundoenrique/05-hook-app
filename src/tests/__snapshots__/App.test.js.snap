// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en componente <App /> Debe mostrarse correctamente el componene <App /> 1`] = `
<Fragment>
  <h1>
    No found
  </h1>
</Fragment>
`;
